agent/sshd/server.go

```diff
@@ -122,13 +123,43 @@ func (s *Server) sessionHandler(session sshserver.Session) {
 
 		os.Chown(pts.Name(), uid, -1)
 
+		hostport := session.RemoteAddr().String()
```

Comment:
`remoteAddr` seems to be a more meaningful variable name.

Reply:
Agreed.